.circleci/build_steps.sh

```diff
@@ -40,7 +40,7 @@ conda clean --lock
 # Ensure that `noarch` exists otherwise `conda` won't think this channel is valid.
 conda index /home/conda/staged-recipes/build_artifacts/noarch
 
-conda install --yes --quiet conda-forge-ci-setup=1.* conda-forge-pinning networkx conda-build
+conda install --yes --quiet conda-forge-ci-setup=1.* conda-forge-pinning networkx conda-build>=3.16
```

Comment:
Why is this needed?

Reply:
It isn’t. Leftover from an old version. Conda smithy will overwrite it anyways when the recipe is merged.

Reply:
Ack. Now I realize that this is probably needed. Conda build 3.15.1 has a bug where it deletes previous versions of packages when it builds matrices of versions but doesn’t rebuild them later. @msarahan fixed this in 3.16+.

Reply:
The deleted packages are the ones not versioned by the build matrix. That would be the executable in this case.

Reply:
We had to put a similar hack into xgboost when updating that feedstock.